lib/convenience_functions.rb

```diff
@@ -0,0 +1,24 @@
+# lib/convenience_functions.rb
+# Extracted bits of crew-specific code that we use frequently enough that it makes sense to split them out to here.
+require 'json'
+require_relative 'const'
+require_relative 'crewlog'
+
+class ConvenienceFunctions
```

Comment:
`JSONFunctions`?

Reply:
I forsee this expanding beyond just json-related functions in the future

Reply:
Well, I don't see the point in creating ConvenienceFunctions and MiscFunctions if both functions will be used generically.  If we are going to organize into separate functions then we need to really refactor with a sensible plan.

Reply:
the purposes of both files are explained at the top-- ConvenienceFunctions is for repeated bits of crew-related code, while MiscFunctions is for implementations of algorithms and conversions and anything else that isnt crew-specific.